travelplan/src/SearchPage/SearchPage.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { TextField, Button, Typography } from "@material-ui/core";
+import SearchIcon from "@material-ui/icons/Search";
+import useStyles from "./SearchPageStyles.js";
+import "./SearchPage.css";
+import { SIGN_IN, SUPPORT } from "./Constants.js";
+
+function SearchPage() {
+  const classes = useStyles();
+
+  return (
+    <div className="search-page">
+      <div className="search-page-buttons">
+        <Button
+          className={`button-style ${classes.buttonStyle}`}
+          variant="outlined"
+          id="main-button-style"
+        >
+          <Typography className="text-button-style">{SIGN_IN}</Typography>
+        </Button>
+        <Button className="button-style">
+          <Typography className="text-button-style">{SUPPORT}</Typography>
+        </Button>
+      </div>
+      <div className="search-page-main">
+        <div className="search-page-assets">
+          <div className="search-page-image"></div>
+          <div className="title">
+            <h1 className="upper-title"> Oompa </h1>
+            <h1 className="bottom-title">Loompas</h1>
```

Comment:
What do you think about moving the imperative values into constants at the outside file like `constants.js`?

From my side, I think that it can be useful to change any imperative data in one place to affect whole application & references